tests/e2e/ios-swift/Podfile

```diff
@@ -1,11 +1,11 @@
-target 'TypewriterSwiftExample' do
-  use_frameworks!
-
-  pod "Analytics"
-  platform :ios, "12.2"
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
 
-  target 'TypewriterSwiftExampleTests' do
-    inherit! :search_paths
-  end
+target 'RudderTyperSwiftExample' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
 
+  # Pods for RudderTyperSwiftExample
+  pod 'Rudder', '1.28.0'
```

Comment:
Don’t fix the Rudder-iOS SDK version; make it flexible up to `2.0.0`.
In case you want to fix the version then please document the steps to update the version in readme.